projects_test.go

```diff
@@ -264,3 +264,106 @@ func TestProjectsService_GetBranch(t *testing.T) {
 		}
 	}
 }
+
+func TestProjectsService_ListAccessRights(t *testing.T) {
```

Comment:
Can we add these tests (tests of functionality inside `projects_access.go`) into a file called `projects_access_test.go`?
This would follow the standard of go that the respective tests last in a similar file.